shared/chat/conversation/index-hoc.js

```diff
@@ -151,6 +151,6 @@ const editLastMessageHoc = compose(
   withProps(({setEditLastMessageCounter}) => ({onEditLastMessage: () => setEditLastMessageCounter(n => n + 1)}))
 )
 
-const hoc = compose(focusInputHoc, editLastMessageHoc, propsHoc, decoratedMesssagesHoc)
+const hoc = compose(focusInputHoc, editLastMessageHoc, decoratedMesssagesHoc, propsHoc)
```

Comment:
so we have to be really careful about the ordering of things in compose. What was happening was the items were being decorated (injecting the supersedes / superseded by stuff) but then it *wasn't* being passed to the children. This fixes that